site/cypress/e2e/pronouns_change.spec.js

```diff
@@ -0,0 +1,68 @@
+
+describe('Tests cases abut changing user pronouns', () => {
+
+    let oldPronouns = '';
```

Comment:
Why is this variable defined here?  If you're trying to handle data unexpected existing before the test suite runs, you might want to take a look at `beforeEach()` and `afterEach()` to handle setting the expected data and then restoring the old data when the tests finish.

Reply:
1) This isn't to handle unexpected data, it's to change the data back
2) These are both changes and tests, if I wrapped it in a before/after call it wouldn't display as a test in cypress. My understanding is before each is usually just for setting the stage for each test, but this requires logging into student and getting the data on a page.
3) If I made it beforeEach/afterEach it would run twice which would slow down the test

Reply:
Would using `before()` and `after()` address point 3?

Reply:
If I did it as `before()` or `after()`, it wouldn't show up as a test, which would be bad as it does test if it changed on the profile page